cli/tests/integrations/test_package.py

```diff
@@ -224,6 +182,34 @@ def test_install_missing_options_file():
         stderr=b"Error opening file [asdf.json]: No such file or directory\n")
 
 
+def test_install_specific_version():
+    stdout = (b'We recommend a minimum of one node with at least 2 '
+              b'CPU\'s and 1GB of RAM available for the Marathon Service.\n'
+              b'Installing package [marathon] version [0.8.1]\n'
+              b'Marathon DCOS Service has been successfully installed!\n\n'
+              b'\tDocumentation: https://mesosphere.github.io/marathon\n'
+              b'\tIssues: https:/github.com/mesosphere/marathon/issues\n\n')
+
+    with _package('marathon',
+                  stdout=stdout,
+                  args=['--yes', '--package-version=0.8.1']):
+
+        returncode_, stdout_, stderr_ = exec_command(
```

Comment:
Ah, well in other tests we just overwrite these variables.  Let's stay consistent.